generators/client/templates/angular/src/main/webapp/app/shared/duration.pipe.ts.ejs

```diff
@@ -6,10 +6,7 @@ import * as moment from 'moment';
   name: 'duration',
 })
 export class DurationPipe implements PipeTransform {
-  transform(value: any): String {
-    if (value) {
-        return moment.duration(value).humanize();
-    }
-    return "";
+  transform(value: any): string {
+    return moment.duration(value).humanize();
```

Comment:
why do you remove the `if (value) {` ?

Reply:
Since it's a primitive type, it can never be null, hence no need to check this anymore !

Reply:
`value` is not a primitive type

Reply:
You are absolutely right, my bad :) Will revert

Reply:
fixed